src/container/components/button/Switch.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react';
+import { View } from 'react-native';
+import styled from 'styled-components';
+import Label from '../text/Label';
+import sizes from '../../../theme/sizes';
+
+interface SwitchProps {
+  children?: Array<ReactElement> | ReactElement;
+  fontSize?: number;
+  color?: string;
+  flex?: number | boolean;
+  text?: string;
+  mt?: number | boolean;
+  mb?: number | boolean;
```

Comment:
Interface might contain intuitive understandable fields, or they should be commented.

Reply:
This is intuitive understandable name of variable.
This style use Material UI.
U can check it https://material-ui.com/
mt = margin-top